app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  validates :email, presence: true, uniqueness: true, email: true
+  validates :password, confirmation: true, length: { in: 6..20 }
+  validates :first_name, :last_name, presence: true
```

Comment:
you can put email first_name and last_name  in the same validation of presence, 
something like this 
`validates :email, :first_name, :last_name, presence: true`and you are missing password too 👀  hahaha.

Reply:
I put the same validation for different fields at one line and I don't forget password this time. 😭 😄